Makefile

```diff
@@ -80,4 +80,5 @@ endif
 	@set -e ;\
 	cd test ;\
 	. venv/bin/activate ;\
+	LD_LIBRARY_PATH=$(PWD)/deps/install/lib \
```

Comment:
Well, it goes like this:
First, RPATH (which we don't set) takes precedence of LD_LIBRARY_PATH.
RUNPATH, on the other hand, gives way for LD_LIBRARY_PATH, which is useful here, because we set redisai.so RUNPATH to $ORIGIN:/opt/redislabs/lib, and since the former is at RedisAI/build/ while the engines libraries are ar RedisAI/deps/install/lib, it will not find them without help.
For that scheme to work with RLTest, I had to fix it not to completely override LD_LIBRARY_PATH with what it thinks best, but rather prepend to it.
Another tricky caveat that might hurt us is the way RPATH may be set in engine libraries (e.g. onnxruntime). If indeed RPATH is set, $ORIGIN is ignored, which may lead to failure. So we need to fix that as well. 
I'm currently validating this scheme in RLEC.